src/Resource/CloudAccount/Endpoint.php

```diff
@@ -100,4 +100,22 @@ protected function _waitForPhpVersion(
       return $entity->get('php_version') === $version;
     };
   }
+
+  /**
+   * Clear Nginx Cache
+   *
+   * @return Entity $this
+   * @throws ResourceException If endpoint not available
+   * @throws ApiException If request fails
+   */
+  public function clearNginxCache() : Entity {
+    $this->_client->request(
+      'POST',
+      self::_URI . "/{$entity->getId()}",
+      ['json' => ['_action' => 'purge-cache']]
+    );
+
```

Comment:
even though there's nothing to wait for in this case, need to use `_wait()` to clear any previously set callback, otherwise `wait()` could possibly wait for the wrong thing:
```php
$this->_wait(null);
```